Mage.Client/src/main/java/mage/client/dialog/NewTournamentDialog.java

```diff
@@ -777,6 +785,24 @@ protected String playerLoadDeck() {
         return "";
     }
 
+    protected String playerLoadJumpstartPacks() {
+        if (fcSelectDeck == null) {
+            fcSelectDeck = new JFileChooser();
+            fcSelectDeck.setAcceptAllFileFilterUsed(false);
+            fcSelectDeck.addChoosableFileFilter(new DeckFileFilter("txt", "Jumpstart Packs (*.txt)"));
```

Comment:
Must use another file chooser. If you init `fcSelectDeck` here then it will be broken for `playerLoad` call (custom cube from deck). Or you can use same `playerLoad` but with additional params like ext, ext description, header.